regression/contracts/variant_multidimensional_nested/test.desc

```diff
@@ -0,0 +1,13 @@
+CORE
+main.c
+--apply-loop-contracts
+activate-multi-line-match
+^\[main.1\].*Check loop invariant before entry: SUCCESS$
+^\[main.2\].*Check that loop invariant is preserved: SUCCESS$
+^\[main.3\].*Check decreases clause on loop iteration: SUCCESS$
+^EXIT=0$
+^SIGNAL=0$
+--
+--
+This test succeeds. It tests whether a multidimensional decreases clause works
+properly. 
```

Comment:
Why is this test case called "nested" though? ...

Reply:
Thanks for spotting this typo! Initially, I had a nested while-loop (i.e. an outer loop and an inner loop) in mind before writing this regression test. That's why I call this regression test "nested." However, I then decided to implement a single-while loop with two index variables, `i` and `j`. I will update the name to a more appropriate one.